src/convertToObject.js

```diff
@@ -12,7 +12,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let arr = sourceString.split(';');
+  const result = {};
+
+  arr = arr.map(i => i.split(':').map(j => j.trim())).filter(i => i.length > 1);
```

Comment:
Maximum allowed line length by most linter configurations - 80.
Try to keep it under 60.
```suggestion
  arr = arr.map(i => i.split(':').map(j => j.trim()))
    .filter(i => i.length > 1);
```